src/encoded/utils.py

```diff
@@ -18,3 +20,20 @@ def get_remote_user(item_with_environ: Union[Request, TestApp]) -> str:
 def get_environ_remote_user(environ: Dict[str, Any]) -> str:
     """Get remote user from associated environment."""
     return environ.get("REMOTE_USER", "")
+
+
+def get_item(
```

Comment:
Reason for not just using this everywhere? We really should never use ff_utils in the portal